src/transformers/babelSyntax.js

```diff
@@ -11,15 +11,16 @@ function cleanStackTrace(message) {
 
 function cleanMessage(message) {
   return message
-  // match until the last semicolon followed by a space
-  // this should match
-  // linux => "(SyntaxError: )Unexpected token (5:11)"
-  // windows => "(SyntaxError: C:/projects/index.js: )Unexpected token (5:11)"
-    .replace(/^Module build failed.*:\s/, 'Syntax Error: ');
+    // match until the last semicolon followed by a space
+    // this should match
+    // linux => "(SyntaxError: )Unexpected token (5:11)"
+    // windows => "(SyntaxError: C:/projects/index.js: )Unexpected token (5:11)"
+    .replace(/^Module build failed.*:\s/, '')
+    .replace(/^ModuleBuildError.*:\s/, '');
```

Comment:
can you add a test for this addition ?

Reply:
Can not create a test for now because of all `dependencies` in this project is too old.
e.g. Webpack & some loaders. Maybe I can try to create with `extract-text-webpack-plugin` instead of `mini-css-extract-plugin`.

Reply:
#90